pdr-admin/src/app/protected-area/protected-area-search/protected-area-search.component.ts

```diff
@@ -68,7 +66,10 @@ export class ProtectedAreaSearchComponent implements OnInit {
 
   submit() {
     if (!this.disableSearch) {
-      this.searchService.fetchData(this.form.value.text);
+      this.form.controls['type'].setValue(this.searchType);
+
+      this.form.value.status.toLowerCase();
```

Comment:
consider `...setValue(this.searchType.toLowerCase())` - otherwise you are setting and resetting the control value.

Alternatively, with the Form Engine you can have the value of a picklist item be different than its display - particularly useful for displaying capitalized items while keeping the value lowercase.